docs/layout.html

```diff
@@ -0,0 +1,107 @@
+{% extends "basic/layout.html" %}
```

Comment:
What is this file doing for us? I tried building the docs without it and I could not tell the difference.  Can we remove it if it's not serving a vital purpose?

Reply:
Similarly to the bootstrap-sphinx.css_t file, this file was edited to fix the sidebar issue. 
Located under C:\Python27\Lib\site-packages\sphinx_bootstrap_theme\bootstrap.
We can discuss options if we prefer to avoid editing these files. 
Note these file were included in the PR to be able to document the changes and transfer it to you,  probably there is a better way to do it though!

Reply:
Ah I see.  So, in order to get this to work, you basically need to copy/paste these files into the installation of the sphinx_bootstrap_theme. Ok, I think there are some better ways to do this.  Just thinking off the top of my head, a better solution would be a python script that edits these files in the installation.  I think, at least in that case, we might be able to incorporate these changes into the auto-generated online docs.  Maybe it's worth a quick search to see what people usually do if they want to edit the sphinx theme. I am sure that we are not the first people to encounter this situation.

Reply:
I like the coding idea a lot, I'll search also for others, approach.

Reply:
There is a built-in solution for customization:

https://sphinx-bootstrap-theme.readthedocs.io/en/latest/README.html#customization

and here is the one for extending layout:

https://sphinx-bootstrap-theme.readthedocs.io/en/latest/README.html#extending-layout-html

Reply:
Thanks @mostaphaRoudsari for the pointers, Yes, having our own template html and custom css files within our source structure and not editing the originals at plugin location it is the much, much better. This is the way to go, working on it next!